src/jstests/formio/components/time.spec.js

```diff
@@ -38,7 +37,7 @@ describe('Time Component', () => {
                 expect(component.error.message).toEqual('invalid_time');
               }
 
-              if (value === invalidValues[2]) {
+              if (value === invalidValues.at(-1)) {
                 done();
```

Comment:
Checking for the last array value to see if we are done is a weird pattern :thinking: I went for `at(-1)` so that you don't have to change the index value each time you add a test case.

Thinking about it, making these tests parametrized sounds better

Reply:
yes, please do parametrize them

Reply:
I could go even further by parameterizing the max/min values, but it might hurt readability (but could remove a lot of duplicated code)